src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -723,6 +758,52 @@ void FancyZones::MoveSizeUpdateInternal(HMONITOR monitor, POINT const& ptScreen,
     }
 }
 
+void FancyZones::HandleVirtualDesktopUpdates(HANDLE fancyZonesDestroyedEvent) noexcept

```

Comment:
Ideally we want this in src/common and have the runner manage the thread.
Given it's not just moving the code there, but it also requires to define the class API, we can leave it for now, but as soon as we merge this into master, please open an issue to track the work to move it to src/common.

Reply:
same could be done to GUID hash specialization